### PR TITLE
Show some errors

### PR DESCRIPTION
**Pull request by niik**
errors-with-stacktraces

Includes stacktraces (when available)

### CODE REVIEW

File: RxSpy.Server/Events/OnErrorEvent.cs [hunk 1]
```diff
@@ -20,6 +21,11 @@ public OnErrorEvent(OperatorInfo operatorInfo, Exception error)
             OperatorId = operatorInfo.Id;
             ErrorType = new TypeInfo(error.GetType());
             Message = error.Message;
+
+            if (error.StackTrace != null && error.StackTrace.Length > 0)
```
Comment (shiftkey) [inline RxSpy.Server/Events/OnErrorEvent.cs]: `!String.IsNullOrWhiteSpace(error.StackTrace)`

File: RxSpy.Server/Observables/OperatorObservable.cs [hunk 2]
```diff
@@ -18,9 +18,9 @@ public OperatorObservable(RxSpySession session, IObservable<T> source, OperatorI
             if (source == null)
                 throw new ArgumentNullException("source");
 
-            // This should never happen
-            if (source is OperatorObservable<T>)
-                throw new ArgumentException("Cannot wrap operator observable in another operator observable");
+            // This should never happen but it does with Start and tasks and whatnot, investigate
```
Comment (shiftkey) [inline RxSpy.Server/Observables/OperatorObservable.cs:21]: :fire: commented out code

File: RxSpy.Server/Events/OnErrorEvent.cs (hunk 1 above)
Comment (niik) [inline RxSpy.Server/Events/OnErrorEvent.cs]: Lol, for some reason I thought `StackTrace` was a list of `StackFrames`. So stoopid

File: RxSpy.Server/Observables/OperatorObservable.cs (hunk 2 above)
Comment (niik) [inline RxSpy.Server/Observables/OperatorObservable.cs:21]: yeah, lemme keep it for a bit though I'll have to revisit this to figure out what the frick is going on

**Comment by niik** [comment]
Oh wait, found another dumb thing I did, hold on one sec

**Pull request merged**